_data/people.yml

```diff
@@ -182,6 +182,19 @@
       icon: github-alt
       name: Github
 
+- name: Rhys MacMillan
+  pic: /img/people/macmillanr.png
+  dates: 2024-Present
+  role: Undergraduate Research Assistant
+  description: 'Ideal ARFC group members seek to advance nuclear energy challenges through advanced computing, physics-informed modeling, and sophisticated simulation'
```

Comment:
Consider taking a look at what other members of the group have written in the description field, it looks like this is the description from the generic group member template.

Reply:
@RhysMacMillan Nathan's comment here is great! Would you mind updating this description to reflect you, rather than ARFC in general?